### PR TITLE
Feat: owner setters functions

### PR DESCRIPTION
**Pull request by haythemsellami**
# Task: owner setters functions

**Should be reviewed/merged after #924**

## Description

Implement onlyOwner setters

Fixes ENG-1372

## Type of change

- [X] New feature
- [ ] Bug fix
- [ ] Testing code
- [ ] Document update or config files

### CODE REVIEW

**Review by alpinechicken — APPROVED** [review]
Lgtm

File: packages/zen-bull-netting/test/unit-test/ZenBullNettingUnit.t.sol [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+    function testSetWithdrawsIndexWhenCallerNotOwner() public {
+        vm.prank(user1);
+        vm.expectRevert(bytes("Ownable: caller is not the owner"));
+        zenBullNetting.setWithdrawsIndex(2);
+    }
+
+    function testSetAuctionTwapPeriod() public {
+        vm.prank(owner);
+        zenBullNetting.setAuctionTwapPeriod(42069);
```
Comment (aleone) [inline packages/zen-bull-netting/test/unit-test/ZenBullNettingUnit.t.sol:105]: lol

**Review by aleone — APPROVED** [review]
lgtm

File: .circleci/config.yml [hunk 2]
```diff
@@ -344,9 +344,10 @@ jobs:
       - run: cd packages/zen-bull-netting && FOUNDRY_PROFILE=fuzz forge test -vv
       
   coverage-zen-bull-netting:
-    working_directory: ~/squeeth/packages/zen-bull
+    working_directory: ~/squeeth/packages/zen-bull-netting
     docker:
-      - image: ghcr.io/foundry-rs/foundry:nightly-e9f274df045d36527eff66f8a6d4e836c7227231
+      # - image: ghcr.io/foundry-rs/foundry:nightly-e9f274df045d36527eff66f8a6d4e836c7227231
```
[the remark this answers is not in the document]
Comment (haythemsellami) [reply .circleci/config.yml:349]: yeah but this is only for this netting project, that commit is still for zen-bull vault

File: packages/zen-bull-netting/src/ZenBullNetting.sol [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    );
+    // @dev OTC price tolerance cannot exceed 20%
+    uint256 public constant MAX_OTC_PRICE_TOLERANCE = 2e17; // 20%
+    uint32 public constant MIN_AUCTION_TWAP = 180 seconds;
+
+    /// @dev owner sets to true when starting auction
+    bool public isAuctionLive;
+
+    /// @dev min WETH amounts to withdraw or deposit via netting
+    uint256 public minWethAmount;
```
[the remark this answers is not in the document]
Comment (haythemsellami) [reply packages/zen-bull-netting/src/ZenBullNetting.sol:26]: Started a thread about this on slack, will do whatever modification in separate PR

**Pull request merged**